src/rust/bitbox02-rust/src/hww/api/bitcoin/signtx.rs

```diff
@@ -2695,8 +2695,8 @@ mod tests {
         let host_nonce = hex!("abababababababababababababababababababababababababababababababab");
         // The host nonce commitment value does not impact this test, but an invalid commitment
         // would fail the antiklepto signature check on the host. The host check is skipped here and
-        // tested in test_keystore_antiklepto.c. That the host nonce was included in the sig is
-        // tested by the siganture fixture test below.x
+        // tested in keystore::tests::test_secp256k1_antiklepto_protocol. That the host nonce was included in the sig is
+        // tested by the siganture fixture test below.
```

Comment:
AI didn't fix `siganture` :D

Reply:
Fixed :D The same typo also exists in secp256k1-zpk (probably also added by me :see_no_evil: ) and one in secp256k1-sys crate haha.